mesheryctl/internal/cli/root/app/app.go

```diff
@@ -69,20 +69,17 @@ func init() {
 func getSourceTypes() error {
 	mctlCfg, err := config.GetMesheryCtl(viper.GetViper())
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrLoadConfig(err)
 	}
 	validTypesURL := mctlCfg.GetBaseMesheryURL() + "/api/application/types"
 	req, err := utils.NewRequest("GET", validTypesURL, nil)
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrCreatingRequest(err)
 	}
 
 	resp, err := utils.MakeRequest(req)
 	if err != nil {
-		utils.Log.Error(err)
-		return nil
+		return utils.ErrRequestResponse(err)
```

Comment:
Revert

Reply:
I will make changes to the returned errors, but to log them i think it best we log the functions
like this    `utils.Log.Error(utils.ErrRequestResponse(err))` instead of `utils.Log.Error(err)`